### DISCUSSION TITLE
wallet connect v2 with wagmi

### DISCUSSION THREAD

**Discussion by ryuk8000** [WalletConnect/walletconnect-monorepo#3608] [Developer Support] [answered]
How to switch wallet if account is already connected and a button click should disconnect it and connect to another account

**Comment by glitch-txs** [comment, marked as answer]
Hi, you can use the wagmi's MetaMask connector or disconnect manually from the wallet extension

https://docs.walletconnect.com/2.0/web3modal/react/wagmi/custom-wallets#custom-wagmi-connectors
https://wagmi.sh/react/connectors/metaMask
